python/whylogs/api/writer/writer.py

```diff
@@ -16,7 +35,12 @@ def check_interval(self, interval_seconds: int) -> None:
         non-negative interval."""
 
     @abstractmethod
-    def write(self, profile: DatasetProfileView, dest: Optional[str] = None) -> None:
+    def write(
+        self,
+        file: Union[Writable, DatasetProfileView],
```

Comment:
We should make `DatasetProfileView <: Writable` so this can just be `file: Writable`